CodeChat_Server/codechat.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Installs CodeChat
+# test
```

Comment:
Update this text. Instead of giving instructions here, just refer to the page where the install instructions are.

Reply:
Which link is best:
https://github.com/bjones1/CodeChat_system/blob/master/CodeChat_Server/install.rst
or
https://codechat-system.readthedocs.io/en/latest/CodeChat_Server/install.html#codechat-server-installation

Reply:
Neither -- instead, create a link to the Linux heading in `install.rst` -- see the [Sphinx docs](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#ref-role). For example, here's a snippet of `install.rst` (currently named `easyinstallnotes.rst`) with an added reST label:

``` rest
Troubleshooting for Windows
^^^^^^^^^^^^^^^^^^^^^^^^^^^

.. figure:: ./docs/Powershell_Error.PNG
    

- If Powershell shows an error informing you that running scripts is forbidden on your system, follow these additional steps
    - Open Powershell in administator mode
    - type ``Set-ExecutionPolicy RemoteSigned``
    - Run Powershell Script again

- If you wish to install your own version of Python make sure you are checking the Add to PATH checkbox on installation

##### I added the line below #####
.. _install CodeChat Server on Linux:

Linux
-----
#.  `Open a terminal <https://www.howtogeek.com/howto/22283/four-ways-to-get-instant-access-to-a-terminal-in-linux/>`__.

#.  Highlight and copy this entire command by pressing Ctrl-C: 
```

To reference this label in `codechat.sh`, use something like:

``` bash
#  To run this script, see the `Linux installation instructions <install CodeChat Server on Linux>`.
```